install_template/config.yaml

```diff
@@ -351,28 +351,34 @@ products:
     platforms:
       - name: CentOS 7
         arch: x86_64
-        supported versions: [3.10, 4.0, 4.1, 4.2, 4.3, 4.4]
+        supported versions: [4.4]
       - name: CentOS 7
         arch: ppc64le
-        supported versions: [3.10, 4.0, 4.1, 4.2, 4.3, 4.4]
+        supported versions: [4.4]
       - name: AlmaLinux 8 or Rocky Linux 8
         arch: x86_64
-        supported versions: [3.10, 4.0, 4.1, 4.2, 4.3, 4.4]
+        supported versions: [4.4]
       - name: RHEL 7 or OL 7
         arch: x86_64
-        supported versions: [3.10, 4.0, 4.1, 4.2, 4.3, 4.4]
+        supported versions: [4.4]
       - name: RHEL 7 or OL 7
         arch: ppc64le
-        supported versions: [3.10, 4.0, 4.1, 4.2, 4.3, 4.4]
+        supported versions: [4.4]
       - name: RHEL 8 or OL 8
         arch: x86_64
-        supported versions: [3.10, 4.0, 4.1, 4.2, 4.3, 4.4]
```

Comment:
Let's try to keep all supported versions here - we can eliminate those we don't want via the deploy mappings.

Reply:
okay, but let me say that it's somewhat painful dealing with the HUNDREDS of files generated when you run a build. I have to visually sift through the results to find the files I'm concerned with. More than once I've realized I'm looking at files that were generated for another version or sometimes even another product! What is the reasoning for keeping those older releases in the config file?